### ISSUE TITLE
Broken for node v5+

### ISSUE THREAD

**Issue by nfischer** [OmarCastro/ShellHive#22]
First off, this is awesome! I was searching around for projects that rely on ShellJS (I'm a maintainer) and I'm really glad I stumbled across this one. I've seen some other PL ideas related to this, but I think this is a really interesting implementation.

I tried this out with node v5 & v6 at first and this didn't work. Running with node v4 works perfectly.

I suspect this is due to how npm 3 flattens out the dependency tree, since the bug involves a broken `require()`:

``` bash
$ npm install # successful
$ bower install # also successful
$ npm start

> ShellHive@0.4.5 start /home/nate/programming/ShellHive
> node app.js

module.js:341
    throw err;
    ^

Error: Cannot find module 'express/node_modules/cookie'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/nate/programming/ShellHive/node_modules/sails/lib/hooks/session/index.js:9:12)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at /home/nate/programming/ShellHive/node_modules/sails/lib/app/configuration/index.js:54:28
    at Function.reduce (/home/nate/programming/ShellHive/node_modules/lodash/dist/lodash.js:3735:25)
    at Configuration.defaultConfig (/home/nate/programming/ShellHive/node_modules/sails/lib/app/configuration/index.js:53:18)
    at Configuration.bound [as defaults] (/home/nate/programming/ShellHive/node_modules/lodash/dist/lodash.js:729:21)
    at Array.async.auto.mixinDefaults (/home/nate/programming/ShellHive/node_modules/sails/lib/app/configuration/load.js:136:41)

npm ERR! Linux 4.2.0-36-generic
npm ERR! argv "/home/nate/.nvm/versions/node/v5.7.1/bin/node" "/home/nate/.nvm/versions/node/v5.7.1/bin/npm" "start"
npm ERR! node v5.7.1
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE
npm ERR! ShellHive@0.4.5 start: `node app.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ShellHive@0.4.5 start script 'node app.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ShellHive package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node app.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs ShellHive
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls ShellHive
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/nate/programming/ShellHive/npm-debug.log
```

I'm running this on Ubuntu 15.10, on both node v5 & v6.

I noticed that `node_modules/express/node_modules/cookie` does **not** exist, but `node_modules/express-session/node_modules/cookie` does in fact exist (so it may be a change in express).

**Comment by OmarCastro** [comment]
Sorry for the late reply, had little time to watch the project until recently.
 I am really glad to hear about that there are more people interested in the project.
Due to difficulty to maintain the project I am planning to change technologies (use Play and Scala.js), since I gained vast amount of experience to use them effectively. And I recently updated the dependencies to use node v6, from 0.10. It was hard work, and I am not sure it works sucessfully beacuse I am trying to host it on the site. I am going to review and update the dependencies again.
